.github/workflows/t-bytecode-compare.yml

```diff
@@ -27,6 +27,16 @@ jobs:
       PLATFORM: ${{ matrix.platform }}
 
     steps:
+      - name: Remove unused software to free up space
+        # macOS runners apparently have > 70 GB free so it's only needed on Ubuntu
+        if: "env.PLATFORM != 'macosx-amd64'"
```

Comment:
Why do you have this extra check?

Reply:
Ah, I'm just blind.

Reply:
But it doesn't hurt removing it, does it?

Reply:
Yeah. Technically, I could leave it on because it does not fail. But it also does not really remove anything (the files are probably not there, though `/usr/` itself exists and has some stuff in it) so I think it's better to make it clear.